tests/smoke/smoke_test.go

```diff
@@ -113,6 +114,33 @@ var _ = Describe("Smoke Tests", func() {
 				cfLogs := cf.Cf("logs", appName, "--recent")
 				return string(cfLogs.Wait().Out.Contents())
 			}, 2*time.Minute, 2*time.Second).Should(ContainSubstring("Console output from test-node-app"))
+		}
+
+		When("app source is < 64k", func() {
+			It("creates a routable app pod in Kubernetes from a source-based app", func() {
+				expectSuccessfulPush("assets/test-node-app")
+			})
+		})
+
+		When("app source is > 64k", func() {
+			var tempdir string
+			BeforeSuite(func() {
```

Comment:
Minor nit: I think this could/should be a `BeforeEach` instead of a `BeforeSuite`.

Reply:
Why? Currently, there is only one test, so it won't make a difference. But if there are multiple `It`s in this `When`-block, then we would create (and delete) multiple (identical) temp directories when using `BeforeEach`. This seems unnecessary to me.